libpromises/generic_agent.h

```diff
@@ -33,6 +33,15 @@
 #define GENERIC_AGENT_CHECKSUM_SIZE ((2*CF_SHA1_LEN) + 1)
 #define GENERIC_AGENT_CHECKSUM_METHOD HASH_METHOD_SHA1
 
+enum output_format
```

Comment:
This will be more or less globally visible, so it should match the value names and be more specific. Although frightfully long, can you rename it to the same name as the enum values below, IOW `generic_agent_config_common_policy_output_format`?